test/kube_testing/k8s.go

```diff
@@ -603,3 +613,30 @@ func (o *K8s) CleanupConfigMaps() {
 		_ = o.clientset.CoreV1().ConfigMaps("default").Delete(cm.Name, &metaV1.DeleteOptions{})
 	}
 }
+
+func (o *K8s) CreateRoles(rolesList ...string) ([]nsmrbac.Role, error) {
```

Comment:
That file becomes kind of big. We might want a separate PR to split it to smaller components.